server/services/stats/side/test.go

```diff
@@ -65,6 +72,30 @@ func (t *testNrClient) GetServersStats() ([]newrelic.Server, error) {
 	return servers, nil
 }
 
+type testTcClient struct {
+	CountGetTask int8
+}
+
+func NewTestTcClient() TcClient {
+	return &testTcClient{
+		CountGetTask: 5,
```

Comment:
but you have inited it in `NewTestConfig` few lines above, why don't just use it?